packages/contracts-ts/scripts/generate-addresses.ts

```diff
@@ -0,0 +1,254 @@
+    introduced: 'Legacy',
+    deprecated: false,
+    proxied: true,
+  },
+  OptimismMintableERC20Factory: {
+    address: '0x4200000000000000000000000000000000000012',
+    introduced: 'Legacy',
+    deprecated: false,
+    proxied: true,
+  },
+  L1BlockNumber: {
+    address: '0x4200000000000000000000000000000000000013',
+    introduced: 'Legacy',
+    deprecated: true,
+    proxied: true,
+  },
+  GasPriceOracle: {
+    address: '0x420000000000000000000000000000000000000F',
+    introduced: 'Legacy',
+    deprecated: false,
+    proxied: true,
+  },
+  GovernanceToken: {
+    address: '0x4200000000000000000000000000000000000042',
+    introduced: 'Legacy',
+    deprecated: false,
+    proxied: false,
+  },
+  L1Block: {
+    address: '0x4200000000000000000000000000000000000015',
+    introduced: 'Bedrock',
+    deprecated: false,
+    proxied: true,
+  },
+  L2ToL1MessagePasser: {
+    address: '0x4200000000000000000000000000000000000016',
+    introduced: 'Bedrock',
+    deprecated: false,
+    proxied: true,
+  },
+  L2ERC721Bridge: {
+    address: '0x4200000000000000000000000000000000000014',
+    introduced: 'Legacy',
+    deprecated: false,
+    proxied: true,
+  },
+  OptimismMintableERC721Factory: {
+    address: '0x4200000000000000000000000000000000000017',
+    introduced: 'Bedrock',
+    deprecated: false,
+    proxied: true,
+  },
+  ProxyAdmin: {
+    address: '0x4200000000000000000000000000000000000018',
+    introduced: 'Bedrock',
+    deprecated: false,
+    proxied: true,
+  },
+  BaseFeeVault: {
+    address: '0x4200000000000000000000000000000000000019',
+    introduced: 'Bedrock',
+    deprecated: false,
+    proxied: true,
+  },
+  L1FeeVault: {
+    address: '0x420000000000000000000000000000000000001a',
+    introduced: 'Bedrock',
+    deprecated: false,
+    proxied: true,
+  },
+} as const
+
+const chains = {
+  1: 'mainnet',
+  10: 'optimism-mainnet',
+  5: 'goerli',
+  420: 'optimism-goerli',
+} as const
+
+if (!glob.sync('node_modules/*').length) {
+  throw new Error(
+    'No node_modules found. Please run `pnpm install` before running this script'
+  )
+}
+
+const nodeModules = path.join(__dirname, '..', 'node_modules')
+
+const bedrockDeployments = {
+  [1]: glob.sync(
+    path.join(nodeModules, '@eth-optimism', 'contracts-bedrock', 'deployments', chains[1], '*.json')
+  ),
+  [10]: glob.sync(
+    path.join(nodeModules, '@eth-optimism', 'contracts-bedrock', 'deployments', chains[10], '*.json')
+  ),
+  [5]: glob.sync(
+    path.join(nodeModules, '@eth-optimism', 'contracts-bedrock', 'deployments', chains[5], '*.json')
+  ),
+  [420]: glob.sync(
+    path.join(nodeModules, '@eth-optimism', 'contracts-bedrock', 'deployments', chains[420], '*.json')
+  ),
+}
+
+// These are here instead of contracts-periphery because contracts-periphery is getting moved to contracts-bedrock eventually so they will be here after that happens
+const peripheryDeployments = {
```

Comment:
perhaps we can get merge https://github.com/ethereum-optimism/optimism/pull/6206 before this so we can delete this temp code and never need to merge it

Reply:
I don't think merging either pr first has any value.  If this merged first then #6206 we can then just do a tiny follow up pr to clean up.

Reply:
contracts periphery no longer exists so we should remove all refs to it in this script